index.js

```diff
@@ -221,8 +221,17 @@ let moxios = {
   * @param {Function} fn The function to execute once waiting is over
   * @param {Number} delay How much time in milliseconds to wait
   */
-  wait: function(fn, delay = this.delay) {
-    setTimeout(fn, delay)
+  wait: function(...args) {
```

Comment:
I think I would do this differently. I would always return a `Promise` for more consistency and then call `then` with `fn` if it was provided:

```js
/**
* Wait for request to be made before proceding.
* This is naively using a `setTimeout`.
* May need to beef this up a bit in the future.
*
* @param {Function} fn The function to execute once waiting is over
* @param {Number} delay How much time in milliseconds to wait
* @return {Promise} A Promise that resolves once the delay is reached
*/
wait: function(...args) {
  let fn = typeof args[0] === 'function' ? args.shift() : null
  let delay = typeof args[0] === 'function' ? args.shift() : this.delay
  let promise = new Promise(resolve => {
    setTimeout(resolve, delay)
  })

  if (typeof fn === 'function') {
    promise.then(fn)
  }

  return promise
}
```